DESCRIPTION

```diff
@@ -46,7 +49,9 @@ Imports:
     tidyr,
     dplyr,
     ape,
-    DirichletMultinomial
+    DirichletMultinomial,
+    ggrepel,
+    SingleCellExperiment
```

Comment:
Is this SingleCellExperiment dependency needed? Is mia dependency enough? --> less dependencies are better

Reply:
I added it as a dep because the CMD check was complaining about `#' @importFrom SingleCellExperiment reducedDim reducedDimNames`, since `reducedDim` and `reducedDimNames` are used in the `.incorporate_rda_vis` helper function.

Is there a better way to do this?

Reply:
Ahh ok, good --> then this is ok, I guess there is no better way to do this

This does not really add dependencies since SCE is already dependency of mia